src/django_oapif/pagination.py

```diff
@@ -23,9 +27,9 @@ def get_paginated_response(self, data):
                         "href": self.get_previous_link(),
                     },
                 ],
-                "numberReturned": len(data["features"]),
+                "numberReturned": numberReturned,
                 "numberMatched": self.count,
-                **data,
+                "data": data,
```

Comment:
are you sure about this one? data is a dictionnary, so it will render differently, no?

Reply:
I think it works as is and that it's unpacked anyway later.

Reply:
yeah maybe revert this too or leave as comment